cas-server-support-jdbc/src/main/java/org/jasig/cas/services/JpaServiceRegistryDaoImpl.java

```diff
@@ -35,6 +37,7 @@
     @PersistenceContext
     private EntityManager entityManager;
 
+    @Transactional(readOnly = false)
```

Comment:
I was thinking about the `JpaTicketRegistry` class which has almost all the sufficient `@Transactional` annotations, not the `JpaServiceRegistryDaoImpl` class. Sorry for the misplaced comment.

However, let's take the `destroyTicketGrantingTicket` method for example:

1. one call to the ticket registry to get the TGT: https://github.com/Jasig/cas/blob/master/cas-server-core/src/main/java/org/jasig/cas/CentralAuthenticationServiceImpl.java#L230
2. one call to the ticket registry to destroy the TGT: https://github.com/Jasig/cas/blob/master/cas-server-core/src/main/java/org/jasig/cas/CentralAuthenticationServiceImpl.java#L233

Am I missing the point?